torcheval/metrics/image/ssim.py

```diff
@@ -9,18 +9,32 @@
 # pyre-ignore-all-errors[16]: Undefined attribute of metric states.
 
 import warnings
+from importlib.util import find_spec
 from typing import Iterable, Optional, TypeVar
 
 import torch
 
-from skimage.metrics import structural_similarity
+if find_spec("torchvision") is not None:
```

Comment:
Shouldn't this be `find_spec("skimage")`?

Reply:
Absolutely, will fix